internal/delegatedSending.js

```diff
@@ -0,0 +1,61 @@
+
+//TODO ordering
+
+module.exports.pools = (contracts, point) => {
+  return contracts.delegatedSending.methods.pools(point).call();
+}
+
+module.exports.getInvited = (contracts, point) => {
+  return contracts.delegatedSending.methods.getInvited(point).call();
+}
+
+module.exports.invitedBy = (contracts, point) => {
+  return contracts.delegatedSending.methods.invitedBy(point).call();
+}
+
+module.exports.canSend = (contracts, as, point) => {
+  return contracts.delegatedSending.methods.canSend(as, point).call();
+}
+
+module.exports.getPool = (contracts, point) => {
+  return contracts.delegatedSending.methods.getPool(point).call();
+}
+
+module.exports.canReceive = (contracts, recipient) => {
+  return contracts.delegatedSending.methods.canReceive(recipient).call();
+}
+
+// NB (jtobin):
```

Comment:
Note to self: we've got enough of these things now that I should look into this comment again at some point.. 😄